dashboard/config/locales/en.yml

```diff
@@ -978,6 +995,10 @@ en:
     csp2_1_2018: 'Unit 2: Digital Information'
     csp3_1_2018: 'Unit 3: Intro to Programming'
     csp4_1_2018: 'Unit 4: Big Data and Privacy'
+    csp-variables: 'Variables'
+    csp-conditionals: 'Conditionals'
+    csp-functions: 'Functions'
+    csp-project: 'Project'
```

Comment:
these should really start with `csp_` (not `csp-`) to follow the convention in this file.

Reply:
🤦‍♀️ oops